### ISSUE TITLE
Token Authentication is blocking access

### ISSUE THREAD

**Issue by mcguiremichael** [acm-uiuc/gpu-cluster#11]
![screenshot from 2018-04-09 18-32-16](https://user-images.githubusercontent.com/22135399/38528323-b1797100-3c24-11e8-8047-509cf935961f.png)

Two days ago it was still open to testing.

**Comment by narendasan** [comment]
the same notebook or a new one?

**Comment by mcguiremichael** [comment]
I just checked to see if I could replicate the issue. When I got that Token Authentication message on the 9th, it happened whenever I clicked on any of the options on the main page, be it Pytorch, Tensorflow, Keras, etc. Token Authentication blocked any further action.

The website is behaving differently now. Now when I click on any of these options, nothing happens. It indicates that the jupyter notebook session is loading, but it never actually loads.

![screenshot from 2018-04-12 10-58-55](https://user-images.githubusercontent.com/22135399/38689180-86a715f4-3e40-11e8-8e87-4167328621fb.png)

This image shows what happened when I attempted to activate PyTorch notebook. It waits forever.
